build.gradle

```diff
@@ -120,7 +120,7 @@ jacocoTestReport {
     }
 }
 
-String version = '5.7.1'
+String version = '5.7.2'
```

Comment:
If this is a breaking change the bump should be 6.0.0. If not then 5.8.0.
Patch releases are only for CVEs or bug fixes. This does not look like a bug but enhancement.
Thanks!

Reply:
It is not breaking I will just bump to 5.8.0 then.
Thanks.